### PR TITLE
🚚 move to `ddev` domain

### PR DESCRIPTION
**Pull request by tyler36**
This PR updates docs for move to `ddev` domain.

### CODE REVIEW

**Comment by rfay** [comment]
There are no tests in ddev-contrib. That's the great thing about the move to add-ons of course.

File: recipes/proxy/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # This is a Dockerfile for use with the ddev-webserver image
 # It should be in .ddev/web-build/Dockerfile
 # and then you can expand on it.
-ARG BASE_IMAGE=drud/ddev-webserver:v1.9.1
+ARG BASE_IMAGE=ddev/ddev-webserver:v1.9.1
```
Comment (rfay) [inline recipes/proxy/Dockerfile:4]: This isn't correct, the dockerhub repo will not yet be updated to use drud. Hopefully we can do that later. Not sure we have control of it.

**Review by rfay — CHANGES_REQUESTED** [review]
It's hard to do things that refer to ddev/ddev at this point, as it hasn't moved. And things that refer to dockerhub can't be changed at this point.

File: recipes/proxy/Dockerfile (hunk 1 above)
Comment (tyler36) [reply recipes/proxy/Dockerfile]: reverted.

**Review by rfay — APPROVED** [review]
Thanks!

**Pull request merged**